lab05/src/mc322/lab05/Peao.java

```diff
@@ -18,15 +20,30 @@ public char presentPiece() {
     		return 'p';
     }
 
-    boolean isValid(){
+    boolean isValid(int []source, int []target, boolean comer){
         boolean valid = false;
-
+        boolean compatible;
+        int y, x;
+        y = Math.abs(source[0] - target[0]);

```

Comment:
No caso do Peão, não acho que devemos utilizar o módulo em y já que essa peça não pode mover para trás. Então, é importante conhecer o sinal para determinar a direção do movimento, se é para frente ou para trás. Além disso, o mover para frente uma peça preta tem sinal diferente de mover para frente uma branca. Podemos verificar a cor e determinar se o sentido é compatível em isCompatible(x, y).
```suggestion
        y = target[0] - source[0];
```